### ISSUE TITLE
Manager 6fb20b3e keeps crashing 

### ISSUE THREAD

**Issue by KaMyKaSii** [topjohnwu/Magisk#3780]
Zenfone 4, Android 9 (no A/B, no SAR, ramdisk in boot partition).
After suffering with stuck in boot logo with the Canary 21405 update, I decided to do a clean install of Magisk for the new 21406 update. Fortunately, Android booted correctly but the Manager keeps crashing every time I try to open it or that some app asks for super user permissions
[managercrashing.txt](https://github.com/topjohnwu/Magisk/files/5882377/managercrashing.txt)

**Comment by yschiebelhut** [comment]
Just to add some experiences that may get useful during investigation:

Also experienced problems with this manager version. There were already problems when updating the manager. Sadly I don't remember the exact message but it said something like "problems reading/parsing managers apk file". Then I did a clean install of the Manager by redownloading the apk manually, but after starting manager, it didn't recognize that the previos version of the Magisk core was installed. Then I reflashed the current Magisk zip in recovery and everything is working fine again.

But there was another problem when rehiding the manager. The app got stuck on the loading circle. When jumping out of it with the home button, there were both the original and the hidden manager visible in the app drawer. After launching the hidden version, the unhidden disappeared though.

**Comment by topjohnwu** [comment]
Uninstall the Magisk app manually and reinstall with a proper one

**Comment by KaMyKaSii** [comment]
> Uninstall the Magisk app manually and reinstall with a proper one

After uninstalling Magisk, I downloaded the latest app-debug.apk, changed the extension to zip and installed it using TWRP. After starting on Android, there is already a "Magisk" app installed, so I thought that I shouldn't install the app-debug.apk manually as an update, that all the necessary steps would already be done by the installer automatically. But I did it now and the Manager went back to work. Thanks
